### ISSUE TITLE
 UserWarning: Seems like `optimizer.step()` has been overridden after learning rate scheduler initialization. Please, make sure to call `optimizer.step()` before `lr_scheduler.step()`. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

### ISSUE THREAD

**Issue by tzhxs** [ThilinaRajapakse/simpletransformers#80]
I get a warning when I do my job, is this vital?

**Comment by ThilinaRajapakse** [comment]
No, it doesn't seem to be having any effect. Not sure why that warning is given either, as `optimizer.step()` is called _after_ `lr_scheduler.step()`.

**Comment by tzhxs** [comment]
thanks!
